v23.1/drop-schedules.md

```diff
@@ -65,6 +63,7 @@ In this example, all schedules with the label `schedule_database` are dropped.
 - [Manage a Backup Schedule](manage-a-backup-schedule.html)
 - [`BACKUP`](backup.html)
 - [`RESTORE`](restore.html)
+- [`CREATE CHANGEFEED`](create-changefeed.html)
```

Comment:
Maybe link the "Create Schedule for Changefeed" table.

Reply:
Done.